experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -331,7 +331,7 @@ export class HttpInstrumentation extends InstrumentationBase<HttpInstrumentation
     request: http.ClientRequest,
     span: Span,
     startTime: HrTime,
-    metricAttributes: MetricAttributes
+    metricAttributes: Attributes
```

Comment:
I wonder if we also need to change param names. But I think that would make it a breaking change.

Reply:
ah, I think that's fine here. The name here is actually helping us understand that the attributes here are intended for the metric data point. :slightly_smiling_face: